vendor/vendor_init.te

```diff
@@ -24,3 +24,5 @@ allow vendor_init proc_irq:file write;
 allow vendor_init proc_kernel_printk:file write;
 
 allow vendor_init persist_file:lnk_file read;
+
+allow vendor_init device:file { create write };
```

Comment:
to do this in right way ... you should label dev/cpuset/foreground/cpus .. and not grant "device" anyway I was searching for this in caf but nothing

Reply:
I tried labeling cpuset, but the problem is that too many AOSP things depend on it being a `device`. You would have to add access to the new label for so many other things. Trying to avoid that. Besides, vendor_init should be trusted.